Sources/MlemMiddleware/FeedLoaders/Post Feed Loaders/SearchPostFeedLoader.swift

```diff
@@ -14,7 +14,9 @@ public class SearchPostFetcher: PostFetcher {
     public var listing: ApiListingType
     
     // setters to allow manual overriding of these for search use cases
-    public func setApi(_ api: ApiClient) { self.api = api }
+    public override func changeApi(to newApi: ApiClient) async {
+        await super.changeApi(to: newApi)
+    }
```

Comment:
Unnecessary?

```suggestion
```

Reply:
This makes `changeApi` public for this loader--it's normally internal because we don't need to do this anywhere else